azurelinuxagent/common/cgroup.py

```diff
@@ -250,53 +288,103 @@ def get_tracked_metrics(self, **kwargs):
         tracked = []
         cpu_usage = self.get_cpu_usage()
         if cpu_usage >= float(0):
-            tracked.append(MetricValue(MetricsCategory.CPU_CATEGORY, MetricsCounter.PROCESSOR_PERCENT_TIME, self.name, cpu_usage))
+            tracked.append(
+                MetricValue(MetricsCategory.CPU_CATEGORY, MetricsCounter.PROCESSOR_PERCENT_TIME, self.name, cpu_usage))
 
         if 'track_throttled_time' in kwargs and kwargs['track_throttled_time']:
             throttled_time = self.get_cpu_throttled_time()
             if cpu_usage >= float(0) and throttled_time >= float(0):
-                tracked.append(MetricValue(MetricsCategory.CPU_CATEGORY, MetricsCounter.THROTTLED_TIME, self.name, throttled_time))
+                tracked.append(
+                    MetricValue(MetricsCategory.CPU_CATEGORY, MetricsCounter.THROTTLED_TIME, self.name, throttled_time))
 
         return tracked
 
 
 class MemoryCgroup(CGroup):
+
+    def _get_memory_stat_counter(self, counter_name, raise_error_if_counter_not_present=True):
```

Comment:
'raise_error_if_counter_not_present' is a little odd... can't the caller just ignore the exception?